### PR TITLE
Added IP_TRANSPARENT option for UDP

### LINKED ISSUE #7871: Add support for IP_TRANSPARENT socket option for UDP connections
Push request at #6835 does not include 'transparent proxy' mode for UDP. setOption() function in EpollDatagramChannelConfig should include IP_TRANSPARENT option.

### PR DESCRIPTION
**Pull request by kubuzetto**
Added IP_TRANSPARENT option for UDP.

TCP support was added with #6835. This fixes #7871.

### CODE REVIEW

File: transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java [hunk 1]
```diff
@@ -371,4 +377,31 @@ public EpollDatagramChannelConfig setReusePort(boolean reusePort) {
...
+        try {
+            datagramChannel.socket.setIpTransparent(ipTransparent);
+            return this;
+        } catch (IOException e) {
+            throw new ChannelException(e);
+        }
+    }
+
 }
+
```
Comment (normanmaurer) [inline transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java]: revert

Comment (kubuzetto) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java]: revert?

Comment (normanmaurer) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java]: Yes please revert this change ...

**Comment by kubuzetto** [comment]
Yes, I did.

File: transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java (hunk 1 above)
Comment (kubuzetto) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java]: Do you mean the entire PR? Why?

Comment (normanmaurer) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java]: No just the one line change above the comment

**Comment by normanmaurer** [comment]
@kubuzetto thanks... merged

**Pull request merged**